tests/Nhs.Appointments.Api.Integration/Scenarios/Booking/ConfirmBookingFeatureSteps.cs

```diff
@@ -71,5 +77,12 @@ public async Task AssertBookingContactDetails(Gherkin.Ast.DataTable dataTable)
         var actualBooking = await Client.GetContainer("appts", "booking_data").ReadItemAsync<BookingDocument>(bookingReference, new Microsoft.Azure.Cosmos.PartitionKey(siteId));
         actualBooking.Resource.ContactDetails.Should().BeEquivalentTo(expectedContactDetails);
     }
+
+    [And("the number of bookings removed should be in the response")]
```

Comment:
Sorry to be a pain, but is there anyway that you could assert in the database that the expected records have actually been removed?